libs/client/ui/src/lib/components/work-card/work-card.component.html

```diff
@@ -37,6 +37,23 @@ <h3 class="text-xl font-medium">
                 <span class="relative -top-1.5">{{ calcApprovalRating(content.stats.likes, content.stats.dislikes) }}%</span>
             </span>
         </div>
+        <ng-container *ngIf="content.tags !== null && content.tags.length > 0">
+            <div class="card-tags flex flex-row items-center">
+                <ng-container *ngIf="content.tags.length === 1; else multipleTags">
+                    <span class="tag tags">{{ content.tags[0] | displayTags }}</span>
+                </ng-container>
+                <ng-template #multipleTags>
+                    <ng-container *ngIf="showAllTags; else showOneTag">
+                        <span class="tag tags">{{ content.tags | displayTags }}</span>
+                        <rmx-icon name="arrow-up-s-line" (click)="toggleShowAllTags()"></rmx-icon>
+                    </ng-container>
+                    <ng-template #showOneTag>
+                        <span class="tag tags">{{ content.tags[0] | displayTags }}</span>
+                        <rmx-icon name="arrow-down-s-line" (click)="toggleShowAllTags()"></rmx-icon>
+                    </ng-template>
+                </ng-template>
+            </div>
+        </ng-container>
```

Comment:
Displays tags in work cards, with button to toggle showing all; displays one by default